src/schemas/json/project.json

```
@@ -265,8 +274,69 @@
               "type": "boolean",
               "description": "Enables/disables concurrent garbage collection.",
               "default": false
+            },
+            "System.GC.HeapCount": {
+              "type": "number",
+              "minimum": 0,
+              "description": "Limits the number of heaps created by the garbage collector."
+            },
```

```suggestion
            "System.GC.HeapCount": {
              "description": "Limits the number of heaps created by the garbage collector.",
              "type": "integer",
              "minimum": 0
            },
```

Optional fix: consistency violation: key order is inconsistent.
Optional fix: `integer` type constraint can be used.